pkg/sql/exec_util.go

```diff
@@ -733,6 +733,14 @@ var overrideAlterPrimaryRegionInSuperRegion = settings.RegisterBoolSetting(
 	false,
 ).WithPublic()
 
+var enableMultipleActivePortals = settings.RegisterBoolSetting(
+	settings.TenantWritable,
+	"sql.defaults.multiple_active_portals.enabled",
```

Comment:
nit: sql.defaults settings are for session variable defaults. instead, can we name this sql.pgwire.multiple_active_portals.enabled? (be sure to update the release note)

and i think this should be public